pennylane/pauli/dla/lie_closure.py

```diff
@@ -136,6 +136,9 @@ def lie_closure(
         >>> dla_ops = [qml.pauli_decompose(op) for op in dla]
 
     """
+    if matrix:
+        return _lie_closure_matrix(generators, max_iterations, verbose, tol)
+
     if not all(isinstance(op, (PauliSentence, PauliWord, np.ndarray)) for op in generators):
```

Comment:
Should arrays still be present in `generators` at this point? :thinking:

Reply:
yep good point, removed 👍